internal/pkg/mapper/orchestrator/path.go

```diff
@@ -0,0 +1,65 @@
+package orchestrator
+
+import "github.com/keboola/keboola-as-code/internal/pkg/model"
+
+// OnObjectPathUpdate - update Phases/Tasks paths.
+func (m *orchestratorMapper) OnObjectPathUpdate(event model.OnObjectPathUpdateEvent) error {
+	if ok, err := m.isOrchestratorConfigState(event.ObjectState); err != nil || !ok {
+		return err
+	}
+
+	// Rename transformation blocks/codes
```

Comment:
Tady má být zřejmě `Rename orchestration phases/tasks`.

Reply:
jj, fixnute v niektorom dalsom PR, dik (uz som to tu neupravoval)